.github/workflows/release.yml

```diff
@@ -1,8 +1,7 @@
 on:
-  workflow_dispatch:
-    workflow: release-sdks
```

Comment:
I thought this would allow me to run this action only but didn't realize it had dependency on the runtime release